WPTT/theme-test-data
# Update themeunittestdata.wordpress.xml with WP 6.1 block markup

This PR adds new posts with blocks generated in WP 6.1.

### How to test

1. Download the file
2. Activate the WordPress Importer tool
3. Import the file, check the box for downloading the media
4. Test all posts, pages and archives. Both the old and the new content should load correctly.
